modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java

```diff
@@ -1372,6 +1377,8 @@ public KBArticle updateStatus(
 			return kbArticle;
 		}
 
+		String action = "add";
```

Comment:
Where possible, can you use already existing constants in the `Constants` class (portal-kernel)? There are at least two that could be used here (ADD and UPDATE).

Reply:
I've followed your advice, thanks